ext/spl/spl_directory.c

```diff
@@ -57,6 +52,13 @@ PHPAPI zend_class_entry *spl_ce_GlobIterator;
 PHPAPI zend_class_entry *spl_ce_SplFileObject;
 PHPAPI zend_class_entry *spl_ce_SplTempFileObject;
 
+// TODO Use standard Error
+#define CHECK_SPL_FILE_OBJECT_IS_INITIALIZED() \
```

Comment:
Maybe accept the `spl_filesystem_object *` as parameter?

Reply:
Good idea, will do that